lepton/image.go

```diff
@@ -230,19 +249,35 @@ func buildImage(c *Config, m *Manifest) error {
 		return errors.Wrap(err, 1)
 	}
 
+	bootImageOpen, err := os.Open(c.Boot)
+	if err != nil {
+		return errors.Wrap(err, 1)
+	}
+	defer bootImageOpen.Close()
+
+	mergedImgPathOpen, err := os.Open(mergedImgPath)
```

Comment:
same as above

Reply:
`c.Boot` and `mergedImgPath` are need to be open to perform `io.Copy` to final image. I thought it's better to rewrite code to use `io.Copy` instead of running shell command `cat`

Reply:
is there a problem with cat? if not  i would keep it as such, using io.Copy buys you nothing, it introduce more code while solving a non-existing problem.

Reply:
Actually there is no problem with `cat` but `io.Copy` is more a "Go way" of working with binary data..

Reply:
I would prefer io.Copy as dependencies on external programs are bad

Reply:
yeh - it might be a bit slower but agreed - we should have as little exec.command as possible